src/pytometry/pp/_process_data.py

```diff
@@ -53,6 +53,7 @@ def find_indexes(
     data_type
         Type of cytometry data
     inplace
```

Comment:
We've slowly converged on not having an inplace parameter but only working with copy. This might go beyond this PR but we disliked having both copy and inplace.